plugins/com.google.cloud.tools.eclipse.javax.annotations/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/org.jsr-305_3.0.1.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src"/>
```

Comment:
How about removing this `src` line? When I checkout this project fresh, the folder doesn't exist, so my Eclipse reports an error that I have to fix. (By either creating an empty directory or removing this line.)